platform/android/java_godot_lib_jni.cpp

```diff
@@ -603,11 +619,37 @@ JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_onRendererPaused(JNIE
 		return;
 	}
 
+#ifdef MODULE_CAMERA_ENABLED
+	CameraServer *camera_server = CameraServer::get_singleton();
+	if (camera_server) {
+		CameraAndroid *camera_android = Object::cast_to<CameraAndroid>(camera_server);
+		if (camera_android) {
+			camera_android->handle_pause();
+		}
+	}
+#endif // MODULE_CAMERA_ENABLED
+
 	if (os_android->get_main_loop()) {
 		os_android->get_main_loop()->notification(MainLoop::NOTIFICATION_APPLICATION_PAUSED);
 	}
 }
 
+#ifdef MODULE_CAMERA_ENABLED
+JNIEXPORT void JNICALL Java_org_godotengine_godot_GodotLib_onScreenRotationChange(JNIEnv *env, jclass clazz) {
```

Comment:
The method should be outside of the `#ifdef`, and be declared in `java_godot_lib_jni.h` like the other jni methods.